app/src/main/java/com/teamagam/gimelgimel/app/viewModels/SendMessageDialogFragmentViewModel.java

```diff
@@ -0,0 +1,106 @@
+
+package com.teamagam.gimelgimel.app.viewModels;
+
+import android.app.Activity;
+import android.content.Context;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.teamagam.gimelgimel.R;
+import com.teamagam.gimelgimel.app.common.logging.LoggerFactory;
+import com.teamagam.gimelgimel.domain.base.logging.Logger;
+import com.teamagam.gimelgimel.presentation.presenters.SendMessagePresenter;
+
+import javax.inject.Inject;
+import butterknife.BindString;
+import butterknife.BindView;
+import rx.Subscription;
+
+
+public class SendMessageDialogFragmentViewModel implements ViewModel {
+
+    private static final String TAG = "SendMessageDialogFragmentViewModel";
+    private Activity activity;
+    protected Logger sLogger = LoggerFactory.create(this.getClass());
+
+    @BindView(R.id.dialog_send_text_message_edit_text)
+    EditText mSendMessageEditText;
+
+    @BindString(R.string.dialog_send_message_invalid_empty_message)
+    String mInvalidMessage;
+
+    @Inject
+    SendMessagePresenter sendMessagePresenter;
+
+    private Subscription subscription;
+
+    private boolean isInputValid(String userMessage) {
+        return !userMessage.isEmpty();
+    }
+    private class handlePresenterView implements SendMessagePresenter.View {
```

Comment:
The view should implements these methods.
The view model shouldn't do UI tasks like toast.